### DISCUSSION TITLE
vscode extension

### DISCUSSION THREAD

**Discussion by joshhopkins** [citycide/tablemark#15] [Ideas]
Nice tool! Any thoughts how this might be used in VS Code / MD-based note taking environment?

The use case I have in mind is referring to a local JSON file with some kind of tag, and import the the markdown output from [tablemark](https://github.com/citycide/tablemark) within a markdown file.

**Comment by haltcase** [comment]
Hey @joshhopkins! I've moved this to a discussion (I just enabled them, so no worries for making an issue).

I haven't considered anything like this and admittedly don't understand the full use case, but would be happy to make/accept any changes here to make something like this possible.

This sounds out of scope for tablemark itself, but I'd think it should already be capable enough to do its part (JSON -> markdown tables) in this larger context.
